### ISSUE TITLE
New Sittuyin .svg piece set

### ISSUE THREAD

**Issue by gbtami** [gbtami/pychess-variants#12]
Burmese people are using his own pieces for Sittuyin (not Makruk pieces).
See https://en.wikipedia.org/wiki/Sittuyin#/media/File:Sittuyin-castle1.jpg or https://www.youtube.com/watch?v=ebRVTzKSP-0
It would be cool to have some stylized .svg burmese sittuyin piece set.

**Comment by CouchTomato87** [comment]
http://myachess.blogspot.com/2012/01/how-to-play-myanmar-traditional-chess-2.html

For future reference

**Comment by CouchTomato87** [comment]
It sounds like several people are interested in playing Sittuyin, so I will add this to my project list.

**Comment by CouchTomato87** [comment]
[Sittuyin.zip](https://github.com/gbtami/pychess-variants/files/3587187/Sittuyin.zip)

And here they are!

**Comment by gbtami** [comment]
Cool! Unfortunately black and white king has something unusual compared to other pieces causing they lose outermost lines somehow. They are OK in both inkscape and my simple image viewer app. Only browsers has this issue. Strange. I committed them anyhow.
![sittuyin](https://user-images.githubusercontent.com/479964/64486330-2012ae80-d22c-11e9-9746-11a3f409267a.png)

**Comment by gbtami** [comment]
https://github.com/gbtami/pychess-variants/blob/master/static/images/pieces/sittuyin/wk.svg
https://github.com/gbtami/pychess-variants/blob/master/static/images/pieces/sittuyin/bk.svg

**Comment by gbtami** [comment]
Btw. I have some ideas to make elephants and knights a little bit better.
1) Add little eye to elephants.
2) Redraw the head of knights to make them little bit more horsy.
 What do you think?

**Comment by CouchTomato87** [comment]
Fixed. They should work now. I have no idea what the problem was. I copied over the same exact path from an older version to replace them and it worked.

As for the animals, those aren't bad ideas. I think there could be a lot to improve the piece set, because it's unnecessarily complicated (I was considering making my own chess-like alternative,, but again -- I don't think it's the worth it at this time). However, this set is commonly used throughout the internet, so I wanted to be completely faithful to the original and make it an exact reproduction. If you want me to change that, that's fine, but to me it's like altering the Merida font or something.
